props/saber_BC_buttons.h

```diff
@@ -0,0 +1,1092 @@
+        previous_preset();
+      }
+      return true;
+    case EVENTID(BUTTON_POWER, EVENT_SECOND_CLICK_LONG, MODE_OFF):
+    if (!mode_volume_) {
+      previous_preset();
+    } else {
+      VolumeDown();
+    }
+    return true;
+
+
+// Enter / Exit Volume MENU
+        
+  #if NUM_BUTTONS == 1
+    case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_OFF | BUTTON_POWER):
+  #else
+    // 2 button
+    case EVENTID(BUTTON_AUX, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+  #endif
+    if (!mode_volume_) {
+      mode_volume_ = true;
+      if (SFX_vmbegin) {
+        hybrid_font.PlayCommon(&SFX_vmbegin);
+      } else {
+        beeper.Beep(0.5, 3000);
+      }
+      STDOUT.println("Enter Volume Menu");
+    } else {
+      mode_volume_ = false;
+      if (SFX_vmend) {
+        hybrid_font.PlayCommon(&SFX_vmend);
+      } else {
+        beeper.Beep(0.5, 3000);
+      }
+      STDOUT.println("Exit Volume Menu");
+    }
+    return true;
+
+
+// Spoken Battery Level
+        
+  #if NUM_BUTTONS == 1
+    case EVENTID(BUTTON_POWER, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
+  #else
+    // 2 and 3 button
+    case EVENTID(BUTTON_AUX, EVENT_FIRST_HELD_LONG, MODE_OFF):
+  #endif
+    // accurate battery level reading every time
+      srand((rand() * 917823) ^ analogRead(batteryLevelPin));
+    // needs this delay
+      delay (500);
+      ::CommandParser::DoParse("play", "battlevel.wav");
+      delay (1900);
+      talkie.SayDigit((int)floorf(battery_monitor.battery()));
+      delay (600);
+      ::CommandParser::DoParse("play", "point.wav");
+      delay (600);
+      talkie.SayDigit(((int)floorf(battery_monitor.battery() * 10)) % 10);
+      talkie.SayDigit(((int)floorf(battery_monitor.battery() * 100)) % 10);
+      delay (1000);
+      ::CommandParser::DoParse("play", "volts.wav");
+    return true;
+
+
+// On Demand Battery Level
+
+  case EVENTID(BUTTON_POWER, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+    if (!mode_volume_) {
+      SaberBase::DoEffect(EFFECT_BATTERY_LEVEL, 0);
+    }
+    return true;
+
+
+
+// Turns Saber ON
+
+  case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_OFF):
+      // No power on without exiting Vol Menu first
+      if (!mode_volume_) {
+        On();
+      }
+    return true;
+
+
+// Activate Muted
+
+  case EVENTID(BUTTON_POWER, EVENT_THIRD_HELD, MODE_OFF):
+    if (!mode_volume_) { 
+      if (SetMute(true)) {
+      unmute_on_deactivation_ = true;
+      On();
+      }
+    }
+    return true;
+
+
+// Stab
+  
+  case EVENTID(BUTTON_NONE, EVENT_THRUST, MODE_ON):
+    SaberBase::DoStab();
+  return true;
+
+
+// Blaster Deflection
+
+    // original sa22c mode if ENABLE_AUTO_SWING_BLAST is not defined
+  #ifndef ENABLE_AUTO_SWING_BLAST
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_SAVED_CLICK_SHORT, MODE_ON):
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_ON):
+              swing_blast_ = false;
```

Comment:
weird indentation

Reply:
Fixed all incorrect indentation throughout the file. Let me know if I missed anything.